panel/src/components/Forms/Blocks/Blocks.vue

```diff
@@ -467,6 +467,16 @@ export default {
       if (this.$el.contains(event.target) === false && (!overlay || overlay.contains(event.target) === false)) {
         this.select(null);
```

Comment:
We should probably return here, right? Or should we continue with the new code even if this `this.select(null)` happened here on line 468?

Reply:
You're probably right, our goal is to empty the selection anyway.